authorization/service/service_test.go

```diff
@@ -68,28 +47,19 @@ func makeConfig(t *testing.T) Config {
 }
 
 func newTestService(t *testing.T) (*Server, func()) {
-	return newTestServiceWithProviders(t, nil, nil)
+	return newTestServiceWithProviders(t)
 }
 
-func newTestServiceWithProviders(t *testing.T, deviceProvider, sdkProvider Provider) (*Server, func()) {
+func newTestServiceWithProviders(t *testing.T) (*Server, func()) {
```

Comment:
Having both `newTestService` and `newTestServiceWithProviders` is no longer necessary